dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml

```diff
@@ -8,7 +8,7 @@
       - @workshop.sessions.each do |session|
         = session.start_date_with_start_and_end_times_us_format
         %br
-  - unless @workshop.location_name.blank? || @workshop.virtual?
+  - unless @workshop.location_name.blank?
```

Comment:
This is a change that affects CSA Capstone but is not specific to it. From now on, we want to show the location regardless, but have the location be the zoom link for virtual workshops.